habitat_baselines/rl/ddppo/policy/running_mean_and_var.py

```diff
@@ -4,18 +4,34 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
+
 import torch
 from torch import Tensor
 from torch import distributed as distrib
 from torch import nn as nn
 
 
+def welford_update(mean, var, count, new_mean, new_var, new_count):
```

Comment:
Let's add docs string with link or description of the algorithm. This looks more like  [Parallel algorithm](https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance#Parallel_algorithm) not Welford one.

Reply:
Yeah, this is welford's parallel mean/var algorithm (which can also be used for running)